changelog.md

```diff
@@ -1,3 +1,13 @@
+# 4.1.3 / 2022-03-01
+
+### Fixed Issues
+
+- Frontend: [#5430](https://github.com/tracim/tracim/issues/5430),
+[#5453](https://github.com/tracim/tracim/issues/5453),
+[#5455](https://github.com/tracim/tracim/issues/5455)
+
+
+
```

Comment:
Remove one line (juste two line necessary here).

Reply:
done, thank  you